modules/cnv-virtctl-commands.adoc

```diff
@@ -42,7 +42,7 @@ the specified port of the node.
 |`virtctl vnc <vmi_name>`
 |Open a VNC connection to a virtual machine instance.
 
-|`virtctl image-upload --dv-name=<datavolume_name> --image-path=</path/to/image>`
+|`virtctl image-upload dv --dv-name=<datavolume_name> --image-path=</path/to/image>`
```

Comment:
No `--dv-name` flag.  So should be:

`virtctl image-upload dv <datavolume_name> --image-path=</path/to/image>`

Reply:
fixed!